cypress/plugins/index.js

```diff
@@ -1,11 +1,14 @@
 
 const wp = require('@cypress/webpack-preprocessor');
-const config = require('../../webpack.config');
+const webpackConfig = require('../../webpack.config');
 
-module.exports = (on) => {
+module.exports = (on, config) => {
   const options = {
-    webpackOptions: config(null, {mode: 'dev'}),
+    webpackOptions: webpackConfig(null, {mode: 'dev'}),
```

Comment:
I had to rename config to webpackConfig since it conflicted with the standard cypress config argument.